MediaBrowser.XbmcMetadata/Savers/SeriesNfoSaver.cs

```diff
@@ -13,26 +14,30 @@ namespace MediaBrowser.XbmcMetadata.Savers
 {
     public class SeriesNfoSaver : BaseNfoSaver
     {
-        protected override string GetLocalSavePath(BaseItem item)
+        public SeriesNfoSaver(
+            IFileSystem fileSystem,
+            IServerConfigurationManager configurationManager,
+            ILibraryManager libraryManager,
+            IUserManager userManager,
+            IUserDataManager userDataManager,
+            ILogger logger)
+            : base(fileSystem, configurationManager, libraryManager, userManager, userDataManager, logger)
         {
-            return Path.Combine(item.Path, "tvshow.nfo");
         }
 
+        /// <inheritdoc />
+        protected override string GetLocalSavePath(BaseItem item)
+            => Path.Combine(item.Path, "tvshow.nfo");
+
+        /// <inheritdoc />
         protected override string GetRootElementName(BaseItem item)
-        {
-            return "tvshow";
-        }
+            => "tvshow";
 
+        /// <inheritdoc />
         public override bool IsEnabledFor(BaseItem item, ItemUpdateType updateType)
-        {
-            if (!item.SupportsLocalMetadata)
-            {
-                return false;
-            }
-
-            return item is Series && updateType >= MinimumUpdateType;
-        }
+            => item.SupportsLocalMetadata && item is Series && updateType >= MinimumUpdateType;
```

Comment:
I think this one is actually correct, but you forgot to invert the boolean for the others.